msteams-platform/toolkit/configure-bot-capability.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Configure-bot-capability
+author: surbhigupta
+description: Learn to configure bot capability within your Teams app.
```

Comment:
Learn how to configure bot capability within your Teams app.

Reply:
updated